regression/cbmc-java/enum1/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
These hopefully don't need to be turned off any more as it shouldn't crash when they are found.